frontend/src/components/pages/Settings.tsx

```diff
@@ -453,7 +619,9 @@ const Settings = (): React.ReactElement => {
           color: "primary.green",
           bgColor: "background.white",
         }}
-        disabled={attemptedSubmit && !isRequestValid()}
+        disabled={
+          !haveSettingsChanged() || (attemptedSubmit && !isRequestValid())
```

Comment:
allow user to save only if settings have changed and the request is valid. allow the user a naive first attempt before disabling invalid requests.

Reply:
nice!